generators/markup/files/page.js

```diff
@@ -0,0 +1,61 @@
+
+import { createElement, PropTypes, Component } from 'react';
+
+function script({ path, content, type, id }, i) {
+  return path ?
+    <script type={type} src={path} key={i}/> :
+    <script type={type} key={i} id={id} dangerouslySetInnerHTML={{
+      __html: content
+    }}/>;
+}
+
+function style({ path, type, content }, i) {
+  return path ?
+    <link rel='stylesheet' href={path} key={i}/> :
+    <style type={type} key={i} dangerouslySetInnerHTML={{
+      __html: content
+    }}/>;
+}
+
+/**
+ * Global 'page' object used to render every page.
+ */
+export default class Page extends Component {
+
+  static propTypes = {
+    markup: PropTypes.string,
+    scripts: PropTypes.array,
+    styles: PropTypes.array,
+    title: PropTypes.string,
+    path: PropTypes.string
+  }
+
+  render() {
+    const {
+      markup = '',
+      scripts = [],
+      styles = [],
+      title = '',
+      path = '/',
+			lang = 'en-US'
+    } = this.props;
+
+    return <html lang={lang}>
+      <head>
+        <meta charSet='utf-8'/>
+        <meta httpEquiv='X-UA-Compatible' content='IE=edge,chrome=1'/>
+        <meta name='viewport'
+					content='width=device-width, initial-scale=1.0, maximum-scale=1'/>
+        <meta name='apple-mobile-web-app-capable' content='yes'/>
+
+        <title>{title}</title>
+				{scripts.filter(script => script.head).map(script)}
```

Comment:
How do we set the `head` property? Webpack provides no affordance for setting custom attributes in the `stats.json` file presently.